src/main/java/org/zalando/pazuzu/feature/FeatureDto.java

```diff
@@ -14,6 +17,17 @@
     @JsonProperty("description")
     private String description;
 
+    public List<Tag> getTags() {
+        return (null != tags) ? tags : Collections.emptyList();
+    }
+
+    public void setTags(List<Tag> tags) {
+        this.tags = tags;
+    }
+
+    @JsonProperty("tags")
+    private List<Tag> tags;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2743746915/issues/source?bid=3131965&fileBranchId=3355916#l29)